src/lib/components/forms/form/form-components/double-input/double-input.component.spec.ts

```diff
@@ -1,24 +1,162 @@
-import { ComponentFixture, TestBed } from '@angular/core/testing';
-
 import { DoubleInputComponent } from './double-input.component';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { HarnessLoader } from '@angular/cdk/testing';
+import { MatInputHarness } from '@angular/material/input/testing';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatInputModule } from '@angular/material/input';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { FormsModule } from '@angular/forms';
+import { L10nTranslationModule } from 'angular-l10n';
+import { MockTooltipDirective } from '../../../../../testing/mock-tooltip.directive';
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { MatFormFieldHarness } from '@angular/material/form-field/testing';
+import { By } from '@angular/platform-browser';
+import { TerraPlacementEnum } from '../../../../../helpers/enums/terra-placement.enum';
+import { DebugElement } from '@angular/core';
+import { TerraRegex } from '../../../../../helpers/regex/terra-regex';
 
+// tslint:disable-next-line:max-function-line-count
 describe('DoubleInputComponent', () => {
     let component: DoubleInputComponent;
     let fixture: ComponentFixture<DoubleInputComponent>;
+    let debugElement: DebugElement;
+    let loader: HarnessLoader;
+    let input: MatInputHarness;
 
     beforeEach(async () => {
-        await TestBed.configureTestingModule({
-            declarations: [DoubleInputComponent]
-        }).compileComponents();
-    });
+        TestBed.configureTestingModule({
+            imports: [MatFormFieldModule, MatInputModule, NoopAnimationsModule, FormsModule, L10nTranslationModule],
```

Comment:
```suggestion
            imports: [MatFormFieldModule, MatInputModule, NoopAnimationsModule, FormsModule],
```
Wird hier nicht genutzt